packages/headless/src/features/configuration/configuration-slice.ts

```diff
@@ -105,6 +105,15 @@ export const configurationReducer = createReducer(
         if (!isNullOrUndefined(action.payload.legacyApiBaseUrl)) {
           state.analytics.legacyApiBaseUrl = action.payload.legacyApiBaseUrl;
```

Comment:
Was discussed live, but for current v2, we cannot do breaking change on the state itself (ie: it's part of the public contract).

So moving the "old" to `legacyApiBaseUrl` cannot be done for v2. We can do it when moving to v3 though.

We can add `nextApiBaseUrl` property and store it there instead

Reply:
Furthermore.... Since there's no "duplex" mode anymore, do we need to store both analytics endpoints in 2 different properties ?  We could just have one I guess ?

Reply:
Do we need to modify anything related to `getOrganizationEndpoints` utility method ?

In other words, if we look at it from the perspective of Atomic (in `index.html` for example) where we try to do this:
```js
await searchInterface.initialize({
      accessToken: '123',
      organizationId: '456',
      organizationEndpoints: await searchInterface.getOrganizationEndpoints('456'),
      analytics: {
             'analyticsMode' : 'next'
       }
});
```

Do we end up with the right properties stored in the state after the configuration reducer has done it's job ?

There's some code that gets executed in `updateBasicConfiguration` switch case which might need to get modified.

Reply:
> Furthermore.... Since there's no "duplex" mode anymore, do we need to store both analytics endpoints in 2 different properties ? We could just have one I guess ?

In case we need to have some cohabitation, and because it's the simplest, I'll go with the nextApiBaseUrl instead of adding conditions on the value of apiBaseUrl.